vrc-get-gui/components/SideBar.tsx

```diff
@@ -66,7 +66,7 @@ export function SideBar({ className }: { className?: string }) {
 				{isDev && <DevRestartSetupButton />}
 				{isDev && (
 					<SideBarItem
-						href={"/settings/palette"}
+						href={"/dev/palette"}
```

Comment:
```suggestion
						href={"/dev-palette"}
```

We might have similar issue when I added another dev page so prefixing dev- is better I think.